cloud_images/centos7/packer.json

```diff
@@ -22,18 +22,50 @@
       {
         "device_name": "/dev/sde",
         "volume_type": "gp2",
-        "volume_size": 20,
+        "volume_size": 50,
         "delete_on_termination": true
       },
       {
         "device_name": "/dev/sdf",
         "volume_type": "gp2",
+        "volume_size": 100,
+        "delete_on_termination": true
+      },
+      {
+        "device_name": "/dev/sdg",
+        "volume_type": "gp2",
+        "volume_size": 50,
+        "delete_on_termination": true
+      },
+      {
+        "device_name": "/dev/sdh",
+        "volume_type": "gp2",
         "volume_size": 20,
         "delete_on_termination": true
+      }
+    ],
+    "launch_block_device_mappings": [
```

Comment:
why do we need both launch launch_block_device_mappings as well as ami_block_device_mappings now?